tests/unit/local/apigw/test_local_apigw_service.py

```diff
@@ -1112,6 +1112,30 @@ def test_custom_content_type_header_is_not_modified(self):
         self.assertIn("Content-Type", headers)
         self.assertEqual(headers["Content-Type"], "text/xml")
 
+    def test_custom_cookies_in_payload_format_version_2(self):
+        lambda_output = (
+            '{"statusCode": 200, "cookies": ["cookie1=test1", "cookie2=test2"], "body": "{\\"message\\":\\"Hello from Lambda\\"}", '
+            '"isBase64Encoded": false}'
+        )
```

Comment:
can we add a test if "Set-Cookie" pre-exists in the headers?

Reply:
Sure! Added that test